packages/jest/src/matchers.ts

```diff
@@ -49,6 +49,31 @@ const findMediaRules = (
   return rules;
 };
 
+const getAllClassNameForms = (
```

Comment:
not yet, ill see how much effort it would take 👀

Reply:
oh blegh supporting `body.selected &:hover` as a filter seems to be annoying to implement, since in `toHaveCompiledCss` we only have information about the *current* element, without any context about what classes parent elements have

Reply:
adding support for `target: '&:hover'` looks doable though, so far it supports `target: ':hover'` (without the &)

Reply:
Oh all good, I figured we'd get a contribution—meant more, did we get that contribution?  I don't care if we don't get contribution to be frank.

My assumption was we'd do like `….replace('&', className + SPECIFICITY)` type of code, but yeah, getting weird/complicated.

Reply:
hmm I don't see any PRs to fix it 🤷 I guess it's not high priority for them either

I've now added support for `&:hover` in the latest commit in this PR. For stuff like `body.selected &:hover`, I created an issue for this https://github.com/atlassian-labs/compiled/issues/1669